site/docs/install-windows.md

```diff
@@ -88,13 +88,14 @@ We recommend installing:
 *   Common MSYS2 packages
 
     You will likely need these to build and run targets that depend on Bash.  MSYS2 does not install
-    these tools by default, so you need to install them manually.
+    these tools by default, so you need to install them manually. Projects that that depend heavily on these tools are for example Tensorflow.
 
     Open the MSYS2 terminal and run this command:
 
     ```
     pacman -S zip unzip patch diffutils git
     ```
+    Optional: If you want to use Bazel from CMD or Powershell make sure to add `MSYS2_INSTALL_PATH/usr/bin` to your `PATH` Environmentvariable. 
```

Comment:
I would remove the above line and add the following at the end of line 98.
```
Projects that depend on Bash tools in `PATH` need this step, for example TensorFlow.
```

Reply:
It actually works if you run Bazel in the MSYS2 Terminal. You just need to add the PATH Variable if you want to use pacman Packages downloaded with MSYS2 in CMD and Powershell

Reply:
This line still sounds a bit strange to me. It's noted as optional, but says make sure to add `MSYS2_INSTALL_PATH/usr/bin` to your `PATH`.

Can we change this to:

Optional: If you want to use Bazel from CMD or Powershell and still be able to use Bash tools, make sure to add `MSYS2_INSTALL_PATH/usr/bin` to your `PATH` Environmentvariable.